src/components/Gallery/Image.js

```diff
@@ -0,0 +1,52 @@
+/* @ts-check */
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ImageModal from './ImageModal';
+
+/**
+ * Image - Small wrapper around an image tag, supports thumbnails
+ *
+ * @example ../../docs/Image.md
+ * @type import('types').Image
+ */
+const Image = ({ image_url, thumb_url, fallback }) => {
+  const [modalIsOpen, setModalOpen] = useState(false);
```

Comment:
good practice to have the state and setState follow the same name
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(false);
```